packages/hub/src/app/users/[username]/models/[slug]/ModelLayout.tsx

```diff
@@ -86,9 +105,33 @@ export const ModelLayout: FC<Props> = ({ username, slug, children }) => {
   const { model } = useLazyLoadQuery<ModelLayoutQuery>(Query, {
     input: { ownerUsername: username, slug },
   });
+  const router = useRouter();
 
   useFixModelUrlCasing(model);
 
+  const dropDown = (
+    <DropdownMenu>
+      <DropdownMenuHeader> Relative Value Functions </DropdownMenuHeader>
+      <DropdownMenuSeparator />
+      {model.currentRevision.relativeValuesExports.map((exportItem) => (
+        <DropdownMenuActionItem
+          key={exportItem.variableName}
+          icon={ScaleIcon}
+          onClick={() =>
+            router.push(
```

Comment:
It's a bit awkward that this is not a <Link> element, wasn't sure what is best here. That said, we don't use Links in other DropdownMenuActionItems

Reply:
We should use Links more, I just didn't get around to implementing `DropdownMenuLinkItem`.

Next.js `Link`s are more performant than `router.push`, because they prefetch on render.

(it's possible to call `router.prefetch` explicitly too, but that's more work)